### PR TITLE
Adds RowsAffected for CopyToWriter and CopyFromReader

### PR DESCRIPTION
**Pull request by nvorobev**
RowsAffected returns the number of rows affected by COPY
https://www.postgresql.org/docs/9.6/sql-copy.html

### CODE REVIEW

**Comment by nvorobev** [comment]
RowsAffected returns the number of rows affected by COPY

**Comment by kirugan** [comment]
@jackc hi! Can you please review this PR. We use pgx at work, and unfortunately for copy we have to use go-pg library instead. This changes allow us to remove usage of go-pg in our project.

**Comment by jackc** [comment]
I like the change. The problem is it technically is a breaking change. Any current use of CopyToWriter and CopyFromReader that checks the error response would no longer compile.

I try pretty hard to follow semver / avoid breaking changes... But on the other hand it is also unreasonable for a straight forward improvement to have to wait months or years for a major release.

Hmm...

Well, I guess I'll chance breaking semver this time and see how much push back there is.
